README.md

```diff
@@ -1,34 +1,134 @@
 # QuietQuality
+There are a lot of different tools that you need to run as you work - possibly before you commit,
+or before you make a pull request, or after you make changes to a class.. style checkers, tests,
+complexity metrics, static analyzers, etc. QuietQuality can make that simpler and faster!
 
-Work in Progress
+Or you may have a huge existing project, that's not _fully_ in compliance with your style guides,
+but you want to avoid introducing _new_ issues, without having to first resolve all of the existing
+ones. QuietQuality can help with that too.
 
-But essentially, QuietQuality is intended for two purposes:
+## Tool Support
+So far, we have support for the following tools:
 
-1. Let you conveniently run tools like rubocop, rspec, and standard against the _relevant_
-   files locally (the files that have changed locally relative to the default branch)
-2. Let you run those tools in CI (probably github actions) and annotate any issues found
-   with _new or modified_ code, without bothering you about existing issues that you didn't
-   touch.
+* rubocop
+* standardrb
+* rspec
+* haml-lint
+* brakeman (though there's no way to run this against only changed files)
 
+Supporting more tools is relatively straightforward - they're implemented by wrapping cli
+invocations and parsing output files (which overall seem to be much more stable interfaces
+than the code interfaces to the various tools), and each tool's support is built orthogonally
+to the others, in a `QuietQuality::Tools::[Something]` namespace, with a `Runner` and a `Parser`.
 
-Basic usage examples:
+## Local Usage Examples
+Working locally, you'll generally want to commit a `.quiet_quality.yml` configuration file into
+the root of your repository - it'll specify which tools to run by default, and how to run them
+(whether you want to only run each tool against the "changed files", whether to filter the
+resulting messages down to only those targeting lines that have been changed), and allows you to
+specify the 'comparison branch', so you don't have to make a request to your origin server every
```

Comment:
Why are we italicizing here? I was using quotes because it was an imprecise phrase (as it's either `--comparison-branch` or `comparison_branch` depending on whether we're in the cli or a configuration file). If we'd like to settle on italics for option names in general, I think that'd be fine.

Reply:
Ah, I think this is connected to the comment below it.